www/resources/views/pages/breakdownTimeline.blade.php

```diff
@@ -0,0 +1,184 @@
+@extends('default')
+
+@section('content')
+
+<?php 
+
+// Copyright 2020 Catchpoint Systems Inc.
+// Use of this source code is governed by the Polyform Shield 1.0.0 license that can be
+// found in the LICENSE.md file.
+
+// TODO: use these in the template
+// $page_keywords = array('Timeline Breakdown','WebPageTest','Website Speed Test','Page Speed');
+// $page_description = "Chrome main-thread processing breakdown$testLabel";
```

Comment:
You can move these  as globals to the .php file (outside the template)

They should just work. If not we'll fix in a follow up diff